frontend/src/components/interface/drawer/NavigationDrawerButtons.vue

```diff
@@ -0,0 +1,30 @@
+<template>
```

Comment:
Ja bym proponował przenieść do tego pliku wszystko związane z przyciskami. Bo teraz tu jest wyświetlanie a tam logika, lepiej by było jakby to było razem. Fajnie to też rozloży wielkość tych plików, bo obecnie tamten ma 174 wiersze, a ten 30.